### PR TITLE
pt-BR: cr2 Day 3 translations

### PR DESCRIPTION
**Pull request by henrif75**
*Stats (with Day 1 and 2 already merged):*
Before: 1869 translated messages, 201 fuzzy translations, 1390 untranslated messages.
After: 2023 translated messages, 164 fuzzy translations, 1273 untranslated messages.
#317 #1463

### CODE REVIEW

File: po/pt-BR.po [hunk 1]
```diff
@@ -5253,23 +5263,25 @@ msgstr ""
...
-msgstr ""
+msgstr "let-else"
 
 #: src/pattern-matching/let-control-flow.md:98
 msgid ""
 "`if-let`s can pile up, as shown. The `let-else` construct supports "
 "flattening this nested code. Rewrite the awkward version for students, so "
 "they can see the transformation."
 msgstr ""
+"`if-let`s podem se acumular, como mostrado. O construção `let-else` suporta "
```
Comment (jcvicelli) [inline po/pt-BR.po]: A construção instead of O

File: po/pt-BR.po [hunk 2]
```diff
@@ -5750,25 +5784,28 @@ msgid ""
 "In the example, `pets` is allocated on the stack and the vector data is on "
 "the heap. The two vector elements are _fat pointers_:"
 msgstr ""
+"No exemplo, `pets` é alocado na pilha e os dados do vetor estão no heap. Os "
```
Comment (jcvicelli) [inline po/pt-BR.po]: should we translate terms like stack?

**Review by jcvicelli — CHANGES_REQUESTED** [review]
LGTM, only 2 small details for check.

File: po/pt-BR.po [hunk 3]
```diff
@@ -7714,6 +7733,11 @@ msgid ""
 "be dereferenced, eliminating use-after-free and other classes of bugs. But, "
 "GC has a runtime cost and is difficult to tune properly."
 msgstr ""
+"Java, Go e Python dependem do coletor de lixo para identificar a memória que "
+"não é mais acessível e descartá-la. Isso garante que qualquer ponteiro possa "
+"ser desreferenciado, eliminando o uso após a liberação e outras classes de "
+"bugs. Mas, o GC tem um custo de tempo de execução e é difícil de ajustar "
```
Comment (jcvicelli) [inline po/pt-BR.po]: GC as in garbage collector, but if we translate previously to coletor de lixo, GC means nothing

File: po/pt-BR.po [hunk 4]
```diff
@@ -8689,27 +8746,29 @@ msgstr ""
...
+msgstr ""
+"O _verificador de empréstimo_ (_borrow checker_) do Rust impõe restrições "
+"sobre as maneiras como você pode emprestar valores. Para um determinado "
+"valor, a qualquer momento:"
 
 #: src/borrowing/borrowck.md:6
-#, fuzzy
 msgid "You can have one or more shared references to the value, _or_"
-msgstr "Você pode ter um ou mais valores `&T` a qualquer momento, _ou_"
+msgstr "Você pode ter um ou mais referências compartilhadas para o valor, _ou_"
```
Comment (jcvicelli) [inline po/pt-BR.po]: uma ou mais

**Review by jcvicelli — DISMISSED** [review]

File: po/pt-BR.po (hunk 2 above)
Comment (henrif75) [reply po/pt-BR.po]: We try to stick as close as possible to the Rust book pt-BR translation: https://rust-br.github.io/rust-book-pt-br/

**Review by jcvicelli — APPROVED** [review]

**Pull request merged**